{"name": "test_information", "status": "broken", "statusDetails": {"message": "UnicodeEncodeError: 'ascii' codec can't encode character '\\u3002' in position 3: ordinal not in range(128)", "trace": "@allure.description(\"信息流接口\")\n    @allure.testcase('http://bug.com/user-login-Lw==.html', name='测试接口')\n    def test_information():\n        # 读取yaml内的数据\n        yaml_data = load_yaml_data.load_data('./yaml_data/yaml_request_data.yaml')\n        # 调用RequestsClass类的post_req方法请求接口\n>       response = RequestsClass().post_req(URL=yaml_data['test_information']['URL'],\n                                            data=json.dumps(yaml_data['test_information']['data']),\n                                            headers=yaml_data['test_information']['headers'])\n\ntest_case\\test_information3.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nconfigfile\\Request_class.py:16: in post_req\n    response = requests.post(url=URL,data=data,headers=headers,**kwargs)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py:119: in post\n    return request('post', url, data=data, json=json, **kwargs)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py:61: in request\n    return session.request(method=method, url=url, **kwargs)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py:530: in request\n    resp = self.send(prep, **send_kwargs)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py:643: in send\n    r = adapter.send(request, **kwargs)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py:439: in send\n    resp = conn.urlopen(\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\urllib3\\connectionpool.py:670: in urlopen\n    httplib_response = self._make_request(\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\urllib3\\connectionpool.py:392: in _make_request\n    conn.request(method, url, **httplib_request_kw)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\http\\client.py:1255: in request\n    self._send_request(method, url, body, headers, encode_chunked)\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\http\\client.py:1296: in _send_request\n    self.putheader(hdr, value)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <urllib3.connection.HTTPConnection object at 0x000001F56A9DADF0>\nheader = 'env。'\nvalues = ('HO9gbqo35qYT91j0fiy/x37Icbkd2UoTnCN2eQ+Y1aFTxFB6Bi3BGI0WurW823fqjOEgvE7ZV1h7Ocf4EZoXBg==',)\n\n    def putheader(self, header, *values):\n        \"\"\"Send a request header line to the server.\n    \n        For example: h.putheader('Accept', 'text/html')\n        \"\"\"\n        if self.__state != _CS_REQ_STARTED:\n            raise CannotSendHeader()\n    \n        if hasattr(header, 'encode'):\n>           header = header.encode('ascii')\nE           UnicodeEncodeError: 'ascii' codec can't encode character '\\u3002' in position 3: ordinal not in range(128)\n\n..\\AppData\\Local\\Programs\\Python\\Python38\\lib\\http\\client.py:1220: UnicodeEncodeError"}, "description": "信息流接口", "attachments": [{"name": "stdout", "source": "d4a340ee-aab7-414e-a7c6-5affe8db0773-attachment.txt", "type": "text/plain"}], "start": 1618481223172, "stop": 1618481223179, "uuid": "aec19291-39c6-4062-9020-6e37cd70ed57", "historyId": "ac7f4079c168e0d169d5df8db56ab99f", "testCaseId": "1f263197ca3f690220fd3c1d66676c2f", "fullName": "test_case.test_information3#test_information", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_information3"}, {"name": "host", "value": "zhangfushuai"}, {"name": "thread", "value": "22704-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_information3"}], "links": [{"type": "test_case", "url": "http://bug.com/user-login-Lw==.html", "name": "测试接口"}]}